components/org.wso2.carbon.identity.api.server.api.resource/org.wso2.carbon.identity.api.server.api.resource.v1/src/gen/java/org/wso2/carbon/identity/api/server/api/resource/v1/APIResourceListItem.java

```diff
@@ -37,6 +40,8 @@ public class APIResourceListItem  {
     private String identifier;
     private String type;
     private Boolean requiresAuthorization;
+    private List<Property> properties = null;
+
```

Comment:
Remove this empty line.

Reply:
Didn't modify the file since this auto generated.